packages/server/.env.example

```diff
@@ -1,4 +1,6 @@
 PORT=3000
+# CORS_ORIGINS="*"
+# EMBEDDING_ORIGINS="*"
```

Comment:
if you search for `DATABASE_PATH`, you might find other places that are using it. We might need to add these new vars to the missing places.

Reply:
Found them. Addressed in 657dace.